docs/spring-boot-runtime/master.adoc

```diff
@@ -13,7 +13,7 @@ include::topics/sb-tomcat-runtime-overview.adoc[leveloffset=+1]
 
 .{SpringBoot} Tested and Verified Version
 
-The {SpringBoot} runtime version `1.5.8` is tested and verified to run with the Embedded Apache Tomcat Container on OpenShift. This embedded container, as well as other components such as the Java container image, is provided as part of a Red Hat subscription when used with {SpringBoot}.
+The {SpringBoot} runtime version `1.5.8.RELEASE` is tested and verified to run with the Embedded Apache Tomcat Container on OpenShift. This embedded container, as well as other components such as the Java container image, is provided as part of a Red Hat subscription when used with {SpringBoot}.
```

Comment:
@lavernw1 you need to use the variable here

Reply:
@rhoads-zach  Are you saying only the suffix is variable

Reply:
You created a variable that currently maps to `1.5.8.RELEASE`. The variable allows us to change this later in a single spot as apposed to having to search the entire repo.

In line 16, you are still hardcoding the value instead of using the variable. Meaning, if you change the variable, the above value never changes.

So instead of 
> The {SpringBoot} runtime version `1.5.8.RELEASE`

it should be
> The {SpringBoot} runtime version `{SpringBootVersion}`

Also, the variable definition is currently:
> :SpringBootVersion: `1.5.8.RELEASE` 

it should be unstyled:
> :SpringBootVersion: 1.5.8.RELEASE